### PR TITLE
Fix config.sample for preview providers

### PR DESCRIPTION
**Pull request by mmattel**
References:
https://github.com/owncloud/core/pull/38807 (Fix config.sample and preview provider)
https://github.com/owncloud/docs/issues/2735 (Document the support for SGI Images)

This is the corresponding `config-to-docs` run.

Backport to 10.7

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc [hunk 1]
```diff
@@ -1329,37 +1329,69 @@ The following providers are enabled by default:
...
+Install the following additional imagick library when using SVG.
+This library adds imagick support for SVG, WMF, OpenEXR, DjVu and Graphviz:
+
+`sudo apt-get install -y libmagickcore-6.q16-3-extra`
+
+Install the `ffmpeg` when using `OC\Preview\Movie` provider.
+
+`sudo apt install -y ffmpeg`
+
+To get a list of file extensions linked to the image provider, change into the owncloud
```
Comment (EParzefall) [inline modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc:1377]:
```suggestion
To get a list of file extensions linked to the image provider, change into the ownCloud
```

**Review by EParzefall — APPROVED** [review]
Just a typo.

File: modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc:1377]: Pls file changes in addition to here also in the referenced core PR.
Then no new `config-to-docs` run is necessary.
Else things get lost on the next run...

**Comment by mmattel** [comment]
> Just a typo.

Also updated core, in sync...

**Review by phil-davis — APPROVED** [review]

**Pull request merged**